src/components/forms/SearchBar/SearchBar.tsx

```diff
@@ -134,6 +152,7 @@ const SearchBar: React.FC<SearchBarProps> = ({
         type="text"
         value={query}
         onChange={onChangeInput}
+        {...(hasSuggestions || { onKeyDown: handleKeyDown })}
```

Comment:
Shouldn't be here `&&`? Please, what do you want to pass? If just simple `onKeyDown={hasSuggestions ? handleKeyDown : noop}` wouldn't be enough

Reply:
the onKeyDown prop is passed only when hasSuggestions is false
`!X && Y` is  the same as `X || Y`

Reply:
Understand 🙂